src/Bicep.Core/Emit/CompileTimeImports/ImportClosureInfo.cs

```diff
@@ -115,7 +115,7 @@ public static ImportClosureInfo Calculate(SemanticModel model)
                     }
                     break;
                 case BicepSynthesizedVariableReference synthesizedVariableRef:
-                    importedVariables.Add(name, new(closure.SymbolsInImportClosure[synthesizedVariableRef], name, synthesizedVariableRef.Value));
+                    importedVariables.Add(name, new(closure.SymbolsInImportClosure[synthesizedVariableRef], name, null, synthesizedVariableRef.Value));
```

Comment:
Is this OK?

Reply:
Even if we were to add a way to re-export imports, I don't think that would apply to synthesized variables (which don't create a symbol in the Bicep model).